server/get_payload.go

```diff
@@ -184,15 +213,25 @@ func (m *BoostService) getPayload(log *logrus.Entry, signedBlindedBeaconBlockByt
 				req.Header.Set(HeaderUserAgent, userAgent)
 
 				// Send the request
-				log.Debug("requesting payload")
+				if version == GetPayloadV1 {
+					log.Debug("requesting payload")
+				} else {
+					log.Debug("requesting payload submission")
+				}
```

Comment:
Could we replace this log message with 
`log.Debug("submitting signed blinded block")`
This way we can avoid the if stmt.